spam/03_spam_data_slicing_tutorial.py

```diff
@@ -362,80 +249,198 @@ def textblob_polarity(x):
 )
 
 # %% [markdown]
-# ## 5. Improve slice performance
+# ## 3. Improve slice performance
 #
 # In classification tasks, we might attempt to increase slice performance with techniques like _oversampling_ (i.e. with PyTorch's [`WeightedRandomSampler`](https://pytorch.org/docs/stable/data.html#torch.utils.data.WeightedRandomSampler)).
 # This would shift the training distribution to over-represent certain minority populations.
-# Intuitively, we'd like to show the model more `short_link` examples so that the representation is better suited to handle them.
+# Intuitively, we'd like to show the model more `short_link` examples so that *the representation is better suited to handle examples in this slice*.
```

Comment:
- Remove "minority"
- This beginning section is too defensive of a way to start this.  You're starting with defense, spending the majority of this valuable intro paragraph on it, and then briefly talking about our approach.  Take the opposite approach: Start by talking about our approach, devote one sentence in there to alternate ones you could try, point to paper